src/algorithms/signal_source/libs/rtl_tcp_dongle_info.cc

```diff
@@ -21,8 +21,6 @@
 #include <string>
 #include <vector>
 
-using boost::asio::ip::tcp;
```

Comment:
this change appears in all of your PRs, and is related to none of them. Are you starting from **next**?

Reply:
Yes. I'm starting from next.
This change is separated into dedicated PR https://github.com/gnss-sdr/gnss-sdr/pull/626 now.